lib/logstash/outputs/elasticsearch/common_configs.rb

```diff
@@ -148,10 +133,6 @@ def self.included(mod)
       # Set max interval between bulk retries.
       mod.config :retry_max_interval, :validate => :number, :default => 2
 
-      # DEPRECATED This setting no longer does anything. If you need to change the number of retries in flight
-      # try increasing the total number of workers to better handle this.
-      mod.config :retry_max_items, :validate => :number, :default => 500, :deprecated => true
```

Comment:
This should also be `:obsolete => "reason ..."` because it is currently deprecated.